packages/specs/src/rpc/beefy.ts

```diff
@@ -0,0 +1,20 @@
+import { RpcModuleSpec } from '@delightfuldot/specs';
+
+export const beefy: RpcModuleSpec = {
+  getFinalizedHead: {
+    docs:
+      'Returns hash of the latest BEEFY finalized block as seen by this client.\n\n' +
+      'The latest BEEFY block might not be available if the BEEFY gadget is not running\n' +
+      'in the network or if the client is still initializing or syncing with the network.\n' +
+      'In such case an error would be returned.',
```

Comment:
We want to put docs lines in an array so the line-break would work.
```suggestion
    docs:
      docs: [
      'Returns hash of the latest BEEFY finalized block as seen by this client.',
      'The latest BEEFY block might not be available if the BEEFY gadget is not running',
      'in the network or if the client is still initializing or syncing with the network.',
      'In such case an error would be returned.',
    ],
```

<img width="985" alt="Screenshot 2024-01-03 at 12 31 54" src="https://github.com/CoongCrafts/delightfuldot/assets/6867026/2dc45c21-cb4d-41ef-b4bb-b7dda09a848d">